Source/IntegrationTests/TestFiles/LitTests/LitTest/ast/function.dfy.expect

```diff
@@ -0,0 +1,15 @@
+function.dfy(7,33): Warning: This ensures clause is part of a bodyless function. Add the {:axiom} attribute to it or the enclosing function to suppress this warning
+function.dfy(7,47): Warning: This ensures clause is part of a bodyless function. Add the {:axiom} attribute to it or the enclosing function to suppress this warning
+function.dfy(8,36): Warning: This ensures clause is part of a bodyless function. Add the {:axiom} attribute to it or the enclosing function to suppress this warning
+function.dfy(8,50): Warning: This ensures clause is part of a bodyless function. Add the {:axiom} attribute to it or the enclosing function to suppress this warning
+
+Dafny program verifier finished with 9 verified, 0 errors
+function.dfy(7,33): Warning: This ensures clause is part of a bodyless function. Add the {:axiom} attribute to it or the enclosing function to suppress this warning
+function.dfy(7,47): Warning: This ensures clause is part of a bodyless function. Add the {:axiom} attribute to it or the enclosing function to suppress this warning
+function.dfy(8,36): Warning: This ensures clause is part of a bodyless function. Add the {:axiom} attribute to it or the enclosing function to suppress this warning
+function.dfy(8,50): Warning: This ensures clause is part of a bodyless function. Add the {:axiom} attribute to it or the enclosing function to suppress this warning
+
+Dafny program verifier finished with 9 verified, 0 errors
+function.dfy(7,9): Error: Function _module._default.Compiled has no body so it cannot be executed
```

Comment:
I think at some point we should adopt something other than 'compiled' here, because I think compiled is not correct. To me compilation means translation to another language, and the situation here is not exactly that. Even if Dafny had an interpreter, you still would not be able to run/execute this function. Also, the function has a contract, so it can be translated to Boogie. In other terms, it can be 'compiled' to Boogie.

Reply:
Anyways, I've changed it all to "compiled"